src/explore-education-statistics-frontend/src/components/__tests__/__snapshots__/Breadcrumbs.test.tsx.snap

```diff
@@ -1,47 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Breadcrumbs renders correctly with just home breadcrumb 1`] = `
-<nav class="govuk-breadcrumbs"
-     aria-label="Breadcrumb"
->
-  <ol class="govuk-breadcrumbs__list"
-      data-testid="breadcrumbs--list"
-  >
-    <li class="govuk-breadcrumbs__list-item">
-      <a href="/"
-         class="govuk-link govuk-breadcrumbs__link"
-      >
-        Home
-      </a>
-    </li>
-  </ol>
-</nav>
-`;
+exports[`Breadcrumbs renders correctly with just home breadcrumb 1`] = `"<nav class="govuk-breadcrumbs" aria-label="Breadcrumb"><ol class="govuk-breadcrumbs__list" data-testid="breadcrumbs--list"><li class="govuk-breadcrumbs__list-item"><a href="/" class="govuk-link govuk-breadcrumbs__link">Home</a></li></ol></nav>"`;
```

Comment:
We need to fix the frontend snapshot tests as all the formatting has been broken by `jest-serializer-html` being commented out (see earlier)